docs/cow-protocol/tutorials/cow-swap/flash-loans.mdx

```diff
@@ -0,0 +1,119 @@
+---
+sidebar_position: 5
+---
+
+# Flash loans
+
+The general setup for a flash loan requires:
```

Comment:
Flashloans are surprisingly nuanced so instead of conveying "this is what you need" I'd suggest "this is the setup for this particular example". This tutorial is not meant to cover the entire solution space but rather one example of how things could work.

Reply:
Agree. I reframed the tutorial to focus entirely on the example.